packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -107,19 +136,24 @@ const initState = (zcf, directorParamManager, debtMint) => {
 
   const vaultParamManagers = makeScalarMap('brand');
 
-  const { publication: metricsPublication, subscription: metricsSubscription } =
-    makeSubscriptionKit();
+  const { metricsPublication, metricsSubscription } =
+    makeMetricsSubscriptionKit(storageNode, marshaller);
```

Comment:
```suggestion
  const { publisher: metricsPublisher, subscriber: metricsSubscriber } =
    makePublisherKit(storageNode, marshaller, 'metrics');
```

Reply:
reading this as fourth https://github.com/Agoric/agoric-sdk/pull/5613/files/33c4924387a82e068418621a29fad091bdb679e5#r903497158